### DISCUSSION TITLE
Grid: FixedHeader="True" prevents background colors from being applied correctly in the header.

### DISCUSSION THREAD

**Discussion by MaMon1963** [vikramlearning/blazorbootstrap#837] [Show and tell]
In an attempt to color the filter row differently from the grid header row, inspired by the transitory solution you kindly provided, I experimented with version v3.0.0-preview.3 on my webassembly application. The result was that the header is colored correctly after setting the "--bb-table-sticky-background-color" parameter to "none", but unfortunately when scrolling the table the header remains fixed but becomes transparent.
Hoping that this can somehow stimulate the solution to the problem, I report here the configuration data and the visual result:

**GRID SETTINGS IN RAZOR PAGE:**
-----------------------------------------------------------------------------------
```
<Grid @ref="grid"
      TItem="ProdottoDTO"
      Class="table table-hover table-bordered table-striped align-middle"
      DataProvider="ProdottoDataProvider"
      FixedHeader="true"
      AllowFiltering="true"
      AllowPaging="true"
      AllowSorting="true"
      AllowSelection="false"
      PageSize="@pageSize"
      FiltersTranslationProvider="SRB_BBCommon.GridFiltersTranslationProvider"
      GridSettingsChanged="OnGridPDSettingsChanged"
      SettingsProvider="GridPDSettingsProvider"
      HeaderRowCssClass="bg-primary text-white border-bottom-0"
      FiltersRowCssClass="bg-success text-white bg-opacity-25 border-bottom-0"
      Height="690"
      EmptyText="Non ci sono dati da mostrare"
      PageSizeSelectorVisible="true"
      PageSizeSelectorItems="null"
      PaginationItemsTextFormat="{0} - {1} di {2} elementi"
      ItemsPerPageText="Elementi per pagina"
      Responsive="true"
      Unit="Unit.Px">
```
-----------------------------------------------------------------------------------
**FILE app.css extraction:**

```
:root {
    --bb-table-sticky-background-color: none;
}
```

**Grid view at first line (even when returning after scrolling down):**
![Screenshot 2024-08-09 234445](https://github.com/user-attachments/assets/9de5d726-c7cb-46b7-a7b3-f1c63bb1699a)

**Grid display when scrolling down...**
**...only the header background (not header fields and related texts) scrolls up...**
![Screenshot 2024-08-10 002346](https://github.com/user-attachments/assets/9e08ffc5-235a-45a4-9fff-af2f1668ba26)

**...then continuing to scroll background disappears:**
![Screenshot 2024-08-09 234528](https://github.com/user-attachments/assets/a2ec6805-8cde-420b-a6ed-593c7f5efa85)

**Comment by mu439** [comment]
@MaMon1963 I'll check this scenario and get back to you.
